helm-charts/konk-service/templates/delete-apiservice-deployment.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: {{ include "konk-service.fullname" . }}-delete-apiservice
+  name: {{ include "konk-service.fullname" . | trunc 43 }}-delete-apiservice
```

This is fine, but it might be a little more obvious truncate the whole string to the actual limit like
```suggestion
  name: {{ printf "%s-delete-apiservice (include "konk-service.fullname" .) | trunc 63 }}
```

---

Never mind, I think I prefer the way you did it originally. A really long name that truncates the entire `-delete-apiservice` part off the end could result in an ambiguous name.

---

One thing I would do either way is trim double `-` like the helpers do:
```suggestion
  name: {{ include "konk-service.fullname" . | trunc 43  | trimSuffix "-" }}-delete-apiservice
```